reference/6/Microsoft.PowerShell.Core/About/about_Providers.md

```diff
@@ -30,142 +30,136 @@ The providers can also add dynamic parameters to the built-in cmdlets. These
 are parameters that are available only when you use the cmdlet with the
 provider data.
 
-## BUILT-IN PROVIDERS
+## Built-in providers
 
-Windows PowerShell includes a set of built-in providers that you can use to
+PowerShell includes a set of built-in providers that you can use to
 access the different types of data stores.
 
 |Provider    |Drive        |Data store                                 |
 |------------|-------------|-------------------------------------------|
-|Alias       |Alias:       |Windows PowerShell aliases                 |
+|Alias       |Alias:       |PowerShell aliases                 |
 |Certificate |Cert:        |x509 certificates for digital signatures   |
 |Environment |Env:         |Windows environment variables              |
 |FileSystem  |(*)          |File system drives, directories, and files |
-|Function    |Function:    |Windows PowerShell functions               |
+|Function    |Function:    |PowerShell functions               |
 |Registry    |HKLM:, HKCU: |Windows registry                           |
-|Variable    |Variable:    |Windows PowerShell variables               |
+|Variable    |Variable:    |PowerShell variables               |
 |WSMan       |WSMan:       |WS-Management configuration information    |
 
 (*) The FileSystem drives vary on each system.
 
-You can also create your own Windows PowerShell providers, and you can install
+You can also create your own PowerShell providers, and you can install
 providers that others develop. To list the providers that are available in
 your session, type:
 
-get-psprovider
-
-## INSTALLING AND REMOVING PROVIDERS
-
-Windows PowerShell providers are delivered to you in Windows PowerShell
-snap-ins, which are .NET Framework-based programs that are compiled into .dll
-files. The snap-ins can include providers and cmdlets.
+```powershell
+Get-PSProvider
+```
 
-Before you use the provider features, you have to install the snap-in and
-then add it to your Windows PowerShell session. For more information, see
-about_PSSnapins.
+## Installing and removing providers
 
-You cannot uninstall a provider, although you can remove the Windows
-PowerShell snap-in for the provider from the current session. If you do,
-you will remove all the contents of the snap-in, including its cmdlets.
+Providers are typically installed via PowerShell modules. Importing the module
```

Comment:
Importing the module does not installing. `Install-Module` installs it. `Import-Module` loads it into your session.

Reply:
done